R/handler_validations.R

```diff
@@ -0,0 +1,152 @@
+#' @title Check Correctness of `data_set`
+#'
+#' @description This auxiliary function checks that the type of the object
+#' provided in `data_set` is a `data.frame` and that the specified columns
+#' are included in it.
+#'
+#' @param data_set `data.frame`.
+#' @param columns A vector of column names that should be present in
+#' `data_set`.
+#'
+#' @return This function does not return a value.
+#'
+#' @keywords internal
+
+check_dataset <- function(data_set, columns) {
```

Comment:
I don't think this needs to be a function. It is only called once in `check_vaccineff_inputs()` so I think you can just call the two asserts directly in `check_vaccineff_inputs()`. This will save you having to maintain the function, it's documentation and tests. Unless you expect to use this function in other places in the future.

Reply:
Right. This will be addressed in https://github.com/epiverse-trace/vaccineff/issues/81